File: Restaurant/C++/Side.cpp
#include "Side.hpp"
#include <string>

Side::Side()
{}

Side::~Side()
{}

Side::Side(std::string name, double price)
{
	this->name = name;
	this->price = price;
}

std::string Side::getName()
{
	return this->name;
}

double Side::getPrice()
{
	return this->price;
}

File: Restaurant/Java/Side.java
package restaurant;

class Side extends Item{
	
	Side(){
	}
	
	Side(String name, double price){
		this.name = name;
		this.price = price;
	}
	
	String getName() {
		return this.name;
	}
	
	double getPrice() {
		return this.price;
	}
	
}